docs/tutorial.md

```diff
@@ -1,3 +1,117 @@
+1. [Cleanup](#cleanup)
+<!-- tocstop -->
+
+## Set up your Backstage app
+
+This tutorial assumes that you have a working Backstage application. To use this tutorial with a local Backstage app, follow the main [Backstage instructions](https://backstage.io/docs/getting-started/create-an-app) to create and run a Backstage app locally.
+
+## Clone this repository
+
+Before we start, clone the repository to get access to the necessary sample code and configurations.
+
+```
+git clone https://github.com/awslabs/backstage-plugins-for-aws.git
+cd backstage-plugins-for-aws
+```
+
+## Create prerequisite AWS resources
+
+To get started, we'll deploy a sample AWS CDK project included in the repository. The CDK project simplifies setup by automatically creating sample ECS services and other resources. These resources allow you to interact with real data in the Backstage plugins without needing to configure everything manually.
+
+### Instructions
+
+1. Navigate to the CDK Project Directory
+
+   Begin by navigating to the directory containing the CDK project:
+
+   ```
+   cd examples/cdk
+   ```
+
+2. Install CDK Project Dependencies
+
+   Inside the directory, install the required dependencies:
+
+   ```
+   yarn install
+   ```
+
+3. Deploy the CDK Stack
+
+   Deploy the sample AWS resources defined in the CDK stack:
+
+   ```
+   cdk deploy
+   ```
+
+   This will provision the AWS resources needed for your project, such as ECS clusters and services.
+
+   Ensure your AWS credentials are configured properly before running the command.
+
+   Deployment may take a few minutes. Once completed, your AWS resources will be ready for use.
+
+## Run the Backstage app locally
+
+With the sample resources in place, you can now run the Backstage app locally to see the plugins in action. The repository already includes pre-installed and configured plugins, so no additional setup is required.
+
+Use the following command to start the Backstage app:
+
+```
+yarn dev
+```
+
+Open your browser and then navigate to `http://localhost:3000/catalog/default/component/example-website`. You should see entities displayed as shown below: ![Example Website](./images/example-website.png)
+
+The `example-website` entity is already loaded from the file system, the Backstage app includes a `entities.yaml` file that defines this entity and its attributes.
+
+This entity is annotated with AWS specific metadata, such as:
+
+```
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: example-website
+  annotations:
+    aws.amazon.com/aws-codepipeline-tags: component=example-website
+    aws.amazon.com/aws-codebuild-project-tags: component=example-website
+    aws.amazon.com/amazon-ecs-service-tags: component=example-website
+    aws.amazon.com/cost-insights-tags: component=example-website
+```
+
+These annotations link the entity to ECS services created by the CDK project, enabling the plugin to fetch and display relevant service details.
```

Comment:
Explain that we are doing this using AWS tags on the associated resources. They can check this in the CDK (provide link).